examples/desktop/misc/simple_event.py

```diff
@@ -0,0 +1,69 @@
+"""
+Simple Event
```

Comment:
This should be much simpler, only one graphic, change the colors, two callbacks one decorated and one not

Reply:
Also add an example of click events, maybe with an image